andela/k2-cfh
# #153009245 Configure ESLint

#### What does this PR do?

Configure ESLint

#### Description of Task to be completed?

- Install ESLint and other dependencies
- Run eslint --init and choose the airbnb popular style guide
- Configure the generated .eslintrc.yml file to include the airbnb style rules

#### How should this be manually tested?

- Clone online repository
- Navigate to the cloned directory
- Add ESLint as extension to your favourite text editor
- Run npm install
- Open the codebase in your favourite text edit to see linting

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

story type: chore
story id: 153009245
story title: Configure ESLint

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2017-11-20 at 4 14 45 pm" src="https://user-images.githubusercontent.com/14099353/33072005-0c31c400-cebe-11e7-8c52-a936b27e2d4b.png">
